CHANGELOG.md

```diff
@@ -1,3 +1,69 @@
+## 1.10.0 (April 09, 2025)
+
+
+## 1.10.0 (April 08, 2025)
```

Comment:
```suggestion
```

Reply:
This is fixed, but GH is having problems. :( 
![image](https://github.com/user-attachments/assets/689ff164-949d-4863-a6fa-e5e3a95e3550)